vpr/src/route/route_common.cpp

```diff
@@ -1026,6 +1032,11 @@ void reset_rr_node_route_structs() {
         route_ctx.rr_node_route_inf[inode].target_flag = 0;
         route_ctx.rr_node_route_inf[inode].set_occ(0);
     }
+
+    for (auto i : device_ctx.rr_node_to_non_config_node_set) {
```

Comment:
I don't think we need to reset this here.  This data should be static throughout an invocation of the router.  So I believe this code can be deleted.
The earlier data structure comments need to describe the variable lifetime, updates (none I think) and use.